plugins/email_service/app/controllers/email_service/configsets_controller.rb

```diff
@@ -1,13 +1,14 @@
 module EmailService
   class ConfigsetsController < ::EmailService::ApplicationController
     before_action :restrict_access
-
+    before_action :check_ec2_creds_cronus_status
```

Comment:
Using the default behaviour you don't need to check here.

Reply:
This is not checking roles anymore. It is checking whether the user has EC2 credentials and nebula enabled or not.